runelite-client/src/test/java/net/runelite/client/chat/ChatMessageManagerTest.java

```diff
@@ -84,4 +85,67 @@ public void onChatMessage()
 
 		verify(messageNode).setValue(eq("<col=b20000>Your dodgy necklace protects you. It has <col=ff0000>1<col=b20000> charge left.</col>"));
 	}
+
+	@Test
+	public void testPublicFriendUsernameRecolouring()
+	{
+		final String localPlayerName = "RuneLite";
+		final String friendName = "Zezima";
```

Comment:
Isn't Zezima a pmod (which would have an icon)?

Reply:
He's been demoted temporarily for this test. Don't worry too much about it.

Reply:
But as a more serious answer though, the names chosen here are arbitrary. Whether players exist with these names or not has no incidence on the tests.